src/web/lib/braze/checkBrazeDependencies.ts

```diff
@@ -0,0 +1,63 @@
+import {
+	checkDependencies,
+	DependencyConfig,
+	DependencyResult,
+} from '@root/src/web/lib/dependencyChecker';
+import { getBrazeUuid } from '@root/src/web/lib/getBrazeUuid';
+import { hasRequiredConsents } from './hasRequiredConsents';
+import { hideSupportMessaging } from './hideSupportMessaging';
+
+const getCompleteDependencies = async (
```

Comment:
the function is using `async`, but you are not using `await` anywhere

Reply:
Good spot, this is fixed in the latest commit I pushed.